torchci/lib/bot/mergeBot.ts

```diff
@@ -138,14 +104,23 @@ function mergeBot(app: Probot): void {
 
     switch (args.command) {
       case "revert":
-        return await handleRevert(args.message);
+        return await dispatchEvent("try-revert", {
+          reason: args.message,
+        });
       case "merge":
-        return await handleMerge(args.force, args.green, args.all_green);
+        return await dispatchEvent("try-merge", {
+          force: args.force,
+          on_green: args.green,
+          all_green: args.all_green,
```

Comment:
i think this will require changes in trymerge.yml b/c it uses -n right now?

Reply:
oh darn

Reply:
Agree, it does not validate truthiness of the statement, just whether option is present or not. But one can probably add function that yanks false arguments